src/Native/pkg/runtime.native.System.Net.Http/runtime.native.System.Net.Http.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   
   <PropertyGroup>
-    <Version>4.0.1</Version>
+    <Version>4.0.2</Version>
```

Comment:
I don't understand the context of these changes.  Is this for the release/1.0.x servicing?  Or is this just making sure we can service stuff moving forward, i.e. .NET Core 1.2 (for which this master branch represents).  And why are these changes just networking specific? Don't other libraries need to be prepared for servicing?